### PR TITLE
Initial pass of Card, Deck, and Hand schemas

### PR DESCRIPTION
**Pull request by clayplumridge**

### CODE REVIEW

File: src/main/resources/json-schema/Deck.schema.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (grundygolem2) [inline src/main/resources/json-schema/Deck.schema.json:1]: does it make sense for this to exist? isn't a deck just a collection of the above schema?

File: src/main/resources/json-schema/Hand.schema.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "$id": "Hand.schema.json",
+  "title": "Hand",
+  "description": "A Magic-Sim hand definition",
+  "type": "object",
+  "properties": {
+    "containsCards": {
```
Comment (grundygolem2) [inline src/main/resources/json-schema/Hand.schema.json:7]: How do we want to handle the logic of, say
1 faithless, 1 enabler?
Does a single faithless support this?
Should name & tag really be separate elements for the purposes of defining a keepable hand?

File: src/main/resources/json-schema/Deck.schema.json (hunk 1 above)
Comment (grundygolem2) [reply src/main/resources/json-schema/Deck.schema.json:1]: Never mind, now I understand what this is for.

**Review by grundygolem2 — COMMENTED** [review]
questions

File: src/main/resources/json-schema/Hand.schema.json (hunk 2 above)
Comment (clayplumridge) [reply src/main/resources/json-schema/Hand.schema.json:7]: For the circumstance of "faithless and any other enabler" ? Eg. Faithless + Shreikhorn?

Comment (grundygolem2) [reply src/main/resources/json-schema/Hand.schema.json:7]: yeah, if we have a hand that is keepable with faithless and 1 other enabler
do we define it as [faithless], [enabler] or as [faithless],[enabler,enabler]

Comment (clayplumridge) [reply src/main/resources/json-schema/Hand.schema.json:7]: I'd suggest that having containsCards: "Faithless" and containsTags: "enabler" means we need at least one Faithless AND one Enabler; when you want a particular card, you're removing that from the tag considerations.

Comment (grundygolem2) [reply src/main/resources/json-schema/Hand.schema.json:7]: either way, this has to be commented and clearly specified

**Review by grundygolem2 — APPROVED** [review]
Good to go!

**Pull request merged**